js/viz/core/export.js

```diff
@@ -330,12 +328,24 @@ _extend(exports.ExportMenu.prototype, {
     },
 
     measure: function() {
+        this._fillSpace();
         return [BUTTON_SIZE + SHADOW_OFFSET, BUTTON_SIZE];
     },
 
     move: function(rect) {
         this._group.attr({ translateX: Math.round(rect[0]), translateY: Math.round(rect[1]) });
     },
+
+    _fillSpace: function() {
+        this._hiddenDueToLayout = false;
+        this._group.attr({ visibility: null });
+    },
+
+    freeSpace: function() {
+        this._incidentOccurred("W2107");
+        this._hiddenDueToLayout = true;
+        this._group.attr({ visibility: "hidden" });
```

Comment:
As _hiddenDueToLayout is still used in code why do you hide export through visibility attribute and not using `hide` method?